src/components/Dashboard/CampaignSummary/index.js

```diff
@@ -50,21 +54,25 @@ export const CampaignSummary = InjectAppServices(({ dependencies: { campaignSumm
           id="dashboard-sentCampaigns"
         />
       </div>
-      <KpiGroup
-        loading={loading}
-        disabled={showOverlay}
-        overlay={
-          <OverlaySection
-            messageKey="dashboard.campaigns.overlayMessage"
-            textLinkKey="dashboard.campaigns.overlayMessageButton"
-            urlKey="dashboard.first_steps.has_campaings_created_url"
-          />
-        }
-      >
-        {kpis.map((kpi) => (
-          <Kpi key={kpi.id} {...kpi} />
-        ))}
-      </KpiGroup>
+      {!hasError ? (
+        <KpiGroup
+          loading={loading}
+          disabled={showOverlay}
+          overlay={
+            <OverlaySection
+              messageKey="dashboard.campaigns.overlayMessage"
+              textLinkKey="dashboard.campaigns.overlayMessageButton"
+              urlKey="dashboard.first_steps.has_campaings_created_url"
+            />
+          }
+        >
+          {kpis.map((kpi) => (
```

Comment:
Does `kpis` have the three results (_My Campaigns_, _My Contacts_, and _First Steps_)?

If I am not wrong, each of them has a different source, so one of them could fail and the other ones succeed.

Is showing the error only in the one that fails possible?

Reply:
Please, disregard my previous comment, reviewing the rest of the code I think that I was wrong.